docx/header.py

```diff
@@ -34,6 +35,13 @@ def is_linked_to_previous(self):
             return True
         return False
 
+    @lazyproperty
+    def body(self):
+        """
+        BlockItemContainer instance with contents of Header
+        """
+        return BlockItemContainer(None, self)
```

Comment:
That's great! Best of luck with the tutorial, I'd love to hear how it goes :)

I won't be going, although I will keep an eye out for next year. I've enjoyed some of the videos from prior years and my wheels are starting to spin now you mention it on how to get work to sponsor me going and maybe participating somehow as a contributor :)